script/Deploy.s.sol

```diff
@@ -23,6 +24,9 @@ contract Deploy is BaseScript {
         sppRepo = _createPluginRepo();
         sppSetup = _createAndCheckNewVersion();
 
+        // Deploy a dummy proxy to force its verification
+        new ERC1967Proxy(address(sppRepo), bytes(""));
```

Comment:
Shouldn't we deploy the proxy with SPP as implementation? to be as close as we could ?
if not it would be better to have a better comment why a proxy is deployed with a repo and what is the aim of it

Reply:
Isn't there already a comment explaining why?